### PR TITLE
Kc/artifacts

### PR DESCRIPTION
**Pull request by jakubczakon**
* added examples for artifacts

### CODE REVIEW

File: how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    return score
+
+
+#
+# Run model training and log dataset version, parameter and test score to Neptune
+#
+
+# Create a Neptune Run and start logging
+# run = neptune.init(project="common/quickstarts",
+#                    api_token="ANONYMOUS")
```
Comment (kamil-kaczmarek) [inline how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py:32]: you can drop this

File: how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+#
+# Change the model parameters
+# Run model training log dataset version, parameter and test score to Neptune
+#
+
+# Create a new Neptune Run and start logging
+# new_run = neptune.init(project="common/quickstarts",
+#                               api_token="ANONYMOUS",
+#                               run=baseline_run_id)
```
Comment (kamil-kaczmarek) [inline how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py:66]: maybe you can drop this

File: how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+          'max_features': 3,
+          }
+new_run["parameters"] = PARAMS
+
+# Calculate the test score and log it to Neptune
+score = train_model(PARAMS, TRAIN_DATASET_PATH, TEST_DATASET_PATH)
+new_run["metrics/test_score"] = score
+
+# Stop logging to the active Neptune Run
+new_run.stop()
```
Comment (kamil-kaczmarek) [inline how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py:106]: you probably want to stop baseline run as well

File: how-to-guides/data-versioning/scripts/run_examples.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+pip install -r requirements.txt
+
+echo "Running Version_datasets_in_model_training_runs.py..."
+python Version_datasets_in_model_training_runs.py
+
+echo "Running Compare_model_training_runs_on_dataset_versions.py..."
+python Compare_model_training_runs_on_dataset_versions.py
+
+# Uncomment the next line to prevent execution window from closing when script execution is complete
+# read -p "Press [Enter] to exit "
```
Comment (kamil-kaczmarek) [inline how-to-guides/data-versioning/scripts/run_examples.sh:11]: Probably you can drop this

File: how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+# Get Neptune Run ID of the first, baseline model training run
+baseline_run_id = run['sys/id'].fetch()
+print(baseline_run_id)
+
+# Stop logging to the active Neptune Run
+run.stop()
+
+#
+# Change the model parameters
```
Comment (kamil-kaczmarek) [inline how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py:59]: Seems that you do this part below in line 94. I guess that it's sort of title for the remaining part, but when I read it for the first time I thought that there is some code missing :)

**Review by kamil-kaczmarek — COMMENTED** [review]
* Left few comments, one is rather important, about closing baseline run,
* I wonder why we have these jpg files in the repo? I did not see that you use them in the code anywhere, as only .csv files are used in the examples.

File: how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py (hunk 3 above)
Comment (jakubczakon) [reply how-to-guides/data-versioning/scripts/Version_datasets_in_model_training_runs.py:106]: Hmm, I thought that if the run is initialized in read-only mode I don't have to stop it.
Just conceptually though, never asked anyone about it.

**Comment by jakubczakon** [comment]
@kamil-kaczmarek resolved all of your comments I think

**Review by kamil-kaczmarek — APPROVED** [review]
awesome :+1:

**Pull request merged**